src/assets/content/CC23.js

```diff
@@ -0,0 +1,412 @@
+export default {
```

Comment:
Long term we should use a similar approach to these charts and the other data viz on the site. This way is good, just will get confusing to have 2 separate tracking files.

Reply:
Maybe moving forward, using this approach so there's a json for each of the lanes on the portfolio (i.e. interactives, chart gallery)

Reply:
yeah, I like that idea!